addons/mrp/migrations/10.0.2.0/pre-migration.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openupgradelib import openupgrade
+from odoo import tools
+
+_column_copies = {
+    'mrp_production': [
+        ('state', None, None)
+    ],
+    'mrp_workorder': [
+        ('state', None, None)
```

Comment:
This field is from module mrp_operations, not mrp.

Reply:
https://github.com/Eficent/OpenUpgrade/pull/11/files

Reply:
But this PR merges both modules before.

Reply:
@pedrobaeza this migration script breaks the migration when mrp_operations was not installed in the first place. I fixed it in my PR on this branch.

Reply:
Ah, OK, got it!